src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -0,0 +1,41 @@
+module sonic-system-aaa {
+    namespace "http://github.com/Azure/sonic-system-aaa";
+    prefix ssys;
+    yang-version 1.1;
+
+    revision 2021-04-15 {
+        description "Initial revision.";
+    }
+
+    container sonic-system-aaa {
+        container AAA {
+            list AAA_LIST {
+                key "type";
+
+                leaf type {
+                    type enumeration {
+                        enum authentication;
+                    }
+                    description "AAA type authentication";
+                }
+
+                leaf login {
```

Comment:
What about defaults ?

Reply:
default is local, will add default login as 'local'

Reply:
Addressed in PR https://github.com/Azure/sonic-buildimage/pull/7671